app/models/tutoring_session.rb

```diff
@@ -2,9 +2,11 @@ class TutoringSession < ApplicationRecord
   belongs_to :student
   belongs_to :tutor
 
-  validates :location,     presence: true
-  validates :start_time,   presence: true
-  validates :end_time,     presence: true
+  validates :location, presence: true
```

Comment:
Roberta doesn't care about location. I put it there because keeping track doesn't hurt. They can just put the city name